.eslintrc

```diff
@@ -8,7 +8,7 @@
     "quotes": 0,
     "consistent-return": 0,
     "no-console": 0,
-    "no-unused-vars": 0,
+    "no-unused-vars": [2, {"vars": "local"}],
```

Comment:
`vars: all` enforces that functions that are exposed also need to be used, which doesn't make much sense.  Especially if they are utility functions.